providers/font-face/FontFaceSessionProvider.js

```diff
@@ -53,6 +64,15 @@ export default class FontFaceSessionProvider extends FontsSessionProvider {
     #parseStyleSheet(styleSheet) {
         for (let i = 0, len = styleSheet.cssRules.length; i < len; i++) {
             let rule = styleSheet.cssRules[i];
+            if (rule.constructor.name === 'CSSImportRule') {
+                let doc = document.implementation.createHTMLDocument(""),
+                  styleElement = document.createElement("style");
+                this.#loadData(rule.href).then((css) => {
+                    styleElement.textContent = css;
+                });
```

Comment:
How can we wait here for the promise to resolve, first?

Reply:
Looks like this did the trick: https://github.com/paulzi/svg-text-to-path/compare/96f2f2db776e33a15584c477ae299e94e0f88ff1..1a56e0b9402e5a87b127187bd564fd45c8fce472